src/panels/lovelace/cards/hui-entity-card.ts

```diff
@@ -124,8 +125,11 @@ export class HuiEntityCard extends LitElement implements LovelaceCard {
         <div class="info">
           <span class="value"
             >${"attribute" in this._config
-              ? stateObj.attributes[this._config.attribute!] ??
-                this.hass.localize("state.default.unknown")
+              ? formatAttributeValue(
+                  this.hass,
+                  stateObj.attributes[this._config.attribute!] ??
+                    this.hass.localize("state.default.unknown")
+                )
```

Comment:
We should not call `formatAttributeValue` on `this.hass.localize("state.default.unknown")`

Reply:
https://github.com/home-assistant/frontend/pull/9252